distribution/docs/src/main/resources/content/_configuring/map-configuration-intrigue.adoc

```diff
@@ -0,0 +1,41 @@
+:title: Map Configuration for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 01
+:summary: Map Configuration for ${catalog-ui}
+
+== {title}
+
+Customize the look of the map displayed to users in ${catalog-ui} through the *Catalog UI Search*.
+Equivalent addition and deletion of a map layer can be found in <<_configuring_map_layers_for_intrigue,Configuring Map Layers for ${catalog-ui}>>.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select the *Catalog UI Search* configuration.
+
+.Edit a Map Layer (Imagery Provider)
+. Enter the properties of the map layer into the *Imagery Provider* in the proper syntax.
+.. Example Imagery Provider Syntax: `{"type": "OSM", "url" "http://a.tile.openstreetmaps.org" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}`.
+... "type": format of imagery provider.
+... "url": location of server hosting the imagery provider.
+... "layers": names of individual layers. (enclose list in square brackets`[ ]`).
```

Comment:
✏️ Change : to . at the end of "parameters": (enclose in braces `{}`):

Reply:
Fixed to delete second colon